Formula/pngquant.rb

```diff
@@ -18,9 +18,9 @@ class Pngquant < Formula
   depends_on "little-cms2"
 
   def install
+    system "./configure", "--prefix=#{prefix}"
     system "make"
```

Comment:
Is the separate `make` step actually needed? The `make install` is probably enough.

Reply:
Habits...